public/js/custom.js

```diff
@@ -0,0 +1,6 @@
+$(".like-button").on('click', function() {
+  $.post('/like', function(data) {
+    console.log('POST request is finished');
+    $(".like-button").text('LIKE COUNT: ' + JSON.parse(data).likes);
```

Comment:
text('LIKE COUNT: ' + JSON.parse(data).likes); is this one not the same as index.js "var likeCount = JSON.parse(fs.readFileSync('likes.json')).likes;"